css/style.css

```diff
@@ -0,0 +1,220 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  background-color: #eee;
+  color: #333;
+  margin: 0;
+  background-image: url("../images/bg-img.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 50% 180%;
+  width: 1349px;
+}
+
+.header-container {
+  background-color: transparent;
+  width: 1320px;
```

Comment:
Setting a fixed width will introduce extra space on its side which will make it stick to the opposite side(in this case to the left.)